python/llm/example/pytorch-models/bert/extract_feature.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+import time
+import argparse
+
+from transformers import BertTokenizer, BertModel
+from bigdl.llm import optimize_model
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Extract the feature of given text using BERT model')
+    parser.add_argument('--repo-id-or-model-path', type=str, default="bert-large-uncased",
+                        help='The huggingface repo id for the BERT (e.g. `bert-large-uncased`) to be downloaded'
+                             ', or the path to the huggingface checkpoint folder')
+    parser.add_argument('--text', type=str, default="What is AI?",
+                        help='Text to extract features')
+
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+
+    # Load model
+    model = BertModel.from_pretrained(model_path)
```

Comment:
shall we also use the options `torch_dtype=auto` and `load_low_cpu_memory_usage=True`?

Reply:
> shall we also use the options `torch_dtype=auto` and `load_low_cpu_memory_usage=True`?

yeah, it would be better if we set it. Have updated.